src/workers/python/init.py.ts

```diff
@@ -1,51 +1,72 @@
+export const INITIALIZE_PYTHON_BACKEND = "__init_papyros";
+export const RUN_PYTHON_CODE = "__run_code";
 export const INITIALIZATION_CODE =
     `
 from pyodide import to_js, eval_code_async
 import sys
+import micropip
+await micropip.install('friendly')
```

Comment:
Beware that this comes with several heavy dependencies. Overall it downloads some 2MB and takes a few seconds. Only starting this after pyodide has initialised is not ideal.

Reply:
I switched to friendly_traceback, which should already cut some of these dependencies. According to https://pyodide.org/en/stable/usage/loading-packages.html there isn't really a way to load them even earlier, or am I mistaken here?

Reply:
I explain what I do in futurecoder here: https://github.com/pyodide/pyodide/issues/1940#issuecomment-985041026

It's significantly more manual, but basically it starts downloading my code and dependencies in parallel with pyodide. Also I can write Python in a normal `.py` file, not in a giant JS string lol.